src/Artsy/Router/buildServerApp.tsx

```diff
@@ -67,6 +67,7 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
             routeConfig: createRouteConfig(routes),
             resolver,
             render,
+            matchContext: { user },
```

Comment:
So dope! @pepopowitz and I were trying to do this exact thing yesterday for a module on the artist page, and were fumbling trying to get the user/lab feature accessible to a route/query. Even with explicitly passing in context from Force we couldn't figure out how to access it inside `prepareVariables`.

So we decided to just make that new component its own query renderer as well, which was sort of a workaround since this exact setup was our initial thought! Although, having that be its own query renderer _might_ actually be a good thing, since it's way below the fold and will likely be an intensive fetch on its own.